src/linear_algebra/matrix/hermitian.lean

```diff
@@ -146,6 +150,16 @@ variables [ring α] [star_ring α] [ring β] [star_ring β]
 
 end ring
 
+section comm_ring
+
+variables [comm_ring α] [star_ring α]
+
+lemma is_hermitian_nonsingular_inv [fintype m] [decidable_eq m] {A : matrix m m α}
+  (hA : A.is_hermitian) : A⁻¹.is_hermitian :=
```

Comment:
Is a similar statement true for `A.adjugate`?

Reply:
Yes, I've added it. The direction `A.adjugate.is_hermitian -> A.is_hermitian` is tricky, though, and I have omitted it.